sdk/maps/azure-maps-render/src/main/java/com/azure/maps/render/models/Center.java

```diff
@@ -0,0 +1,34 @@
+package com.azure.maps.render.models;
+
+import java.util.List;
+import java.util.Objects;
+
+public class Center {
+    private LatLong coordinates;
```

Comment:
remove comments.

just checking: this is n object that will be passed to the client, right? not returned